projects/js-packages/components/components/scan-report/index.tsx

```diff
@@ -152,18 +153,25 @@ export default function ScanReport( { dataSource, data, onChangeSelection } ): J
 				id: FIELD_TYPE,
 				label: __( 'Type', 'jetpack-components' ),
 				elements: TYPES,
+				enableHiding: false,
 			},
 			{
 				id: FIELD_NAME,
 				label: __( 'Name', 'jetpack-components' ),
+				enableHiding: false,
 				enableGlobalSearch: true,
 				getValue( { item }: { item: ScanReportExtension } ) {
+					if ( view.type === 'list' && item.type === 'files' ) {
+						return 'Files';
+					}
+
```

Comment:
```suggestion
```

Ideally, we shouldn't be worrying about something like this at the component level - we should expect that the necessary data is provided.

Luckily, since we control the `files` item, can we avoid needing this side effect in the first place by adding `name: __( 'Files', 'jetpack-components' )` when providing the data to the component on the implementation side in Protect. :+1:

Reply:
I've included adding the `name` property in this PR: https://github.com/Automattic/jetpack/pull/40616

Reply:
>Ideally, we shouldn't be worrying about something like this at the component level - we should expect that the necessary data is provided.

The issue I was attempting resolve was that in list view the Files entry in the table is no obvious what it refers to:
![Screen Shot on 2024-12-16 at 12-35-18](https://github.com/user-attachments/assets/bcd2e7f6-09a2-4521-ab1d-5448792e4389)

The issue with adding `Files` for the files object `name` prop is that it now shows up twice in table view:
![Screen Shot 2024-12-16 at 12 30 28](https://github.com/user-attachments/assets/a248e678-cfc2-401f-84f2-cb74f250b3ec)

Regardless of approach, it appears were able to still display it twice in list view by manually adding the other field with the duplicate string 🤔 Maybe as a third alternative, we could just display `Type` by default in list view?